types/dygraphs/index.d.ts

```diff
@@ -1232,52 +1490,51 @@ declare class Dygraph {
      *
      * This is far more efficient than destroying and re-instantiating a
      * Dygraph, since it doesn't have to reparse the underlying data.
-     */
-    resize(): void;
-
-    /**
-     * Resizes the dygraph. If no parameters are specified, resizes to fill the
-     * containing div (which has presumably changed size since the dygraph was
-     * instantiated. If the width/height are specified, the div will be resized.
+     * {@link https://dygraphs.com/jsdoc/symbols/Dygraph.html#resize}
      *
-     * This is far more efficient than destroying and re-instantiating a
-     * Dygraph, since it doesn't have to reparse the underlying data.
-     *
-     * @param {number} width Width (in pixels)
-     * @param {number} height Height (in pixels)
+     * @param width Width (in pixels)
+     * @param height Height (in pixels)
      */
     resize(width: number, height: number): void;
+    resize(): void;
 
     /**
      * Adjusts the number of points in the rolling average. Updates the graph to
      * reflect the new averaging period.
-     * @param {number} length Number of points over which to average the data.
+     * {@link https://dygraphs.com/jsdoc/symbols/Dygraph.html#adjustRoll}
+     *
+     * @param length Number of points over which to average the data.
      */
     adjustRoll(length: number): void;
 
     /**
      * Returns a boolean array of visibility statuses.
+     * {@link https://dygraphs.com/jsdoc/symbols/Dygraph.html#visibility}
      */
     visibility(): boolean[];
 
     /**
-     * Changes the visiblity of a series.
+     * Changes the visibility of one or more series.
+     * {@link https://dygraphs.com/jsdoc/symbols/Dygraph.html#setVisibility}
      *
-     * @param {number} num the series index
-     * @param {boolean} value true or false, identifying the visibility.
+     * @param num the series index
+     * @param value true or false, identifying the visibility.
      */
-    setVisibility(num: number, value: boolean): void;
+    setVisibility(num: number | number[] | object, value: boolean): void;
```

Comment:
The jsdoc comment here needs to be updated to explain the more complex type on the `num` param:

```ts
/**
 * @param num the series index or an array of series indices
 *            or a boolean array of visibility states by index
 *            or an object mapping series numbers, as keys, to
 *            visibility state (boolean values)
 */
```

Reading that comment, the `object` form should be `{[seriesIndex: number]: boolean}`.

Reply:
Thanks, I will update it.